src/sdk/model/model.ts

```diff
@@ -39,11 +39,19 @@ export type Output = {
     readonly source: string
 };
 
+export type CodeSnippetContext = {
+    readonly language: string;
+    readonly snippet: string;
+    readonly relativePath: string;
+    readonly score: number;
```

Comment:
How we will get the score on the client?

Reply:
Will use [Jaccard_index](https://en.wikipedia.org/wiki/Jaccard_index) algorithms, each snippet will have a score it's pretty much a simple version of cosine_similarity for vectors in the embedding concept